repldex/backend/templates/base.html

```diff
@@ -1,53 +1,33 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
 {% block head %}
 <title>{% block title %}Repldex{% endblock %}</title>
+
 <link rel="icon" type="image/png" href="/static/icon.png">
-<link rel="stylesheet" href="/static/main.css">
+<link rel="preload" href="/static/main.css" as="style">
+<link rel="preload" href="/static/lazy-images.js" as="script">
```

Comment:
Why move this to a different file?

Reply:
It's used in almost every route, I thought preloading it would be better than inline JS for every page.